querybook/webapp/redux/dataDoc/action.ts

```diff
@@ -621,3 +624,43 @@ export function rejectDataDocAccessRequest(
         }
     };
 }
+
+export function getDataDocWithSchema({
```

Comment:
don't use dataDoc redux since it is already bloated with dataDoc actions, can you make a new redux action/reducer called scheduledDataDoc or taskSchedule?